src/main/webapp/about.jsp

```diff
@@ -66,5 +66,25 @@
       </p>
     </div>
   </div>
+  
+<div id="container">
+	<div
```

Comment:
Make sure to check out the html style guide here:   https://google.github.io/styleguide/htmlcssguide.html
There's a bit of indentation mismatches here.

Reply:
I will take a look, thank you! It is totally new for me to work with html so I might make a lot of mistakes.